eng/pipelines/sdk-update.yml

```diff
@@ -190,7 +190,6 @@ stages:
       - job: Create_PR
         dependsOn: 
           - Update_Generator_Versions
-          - Update_Code
```

Comment:
Are those pipeline changes related to the support of projected name?

Reply:
No. it is not related. It is to temporary solution to generate pr even the codegen fail. So we can bypass the openai customize code failure to continue the process.